av1an-core/src/ffmpeg.rs

```diff
@@ -174,6 +174,8 @@ pub fn encode_audio<S: AsRef<OsStr>>(
       "0",
       "-map",
       "-0:a",
+      "-map",
+      "-0:d",
```

Comment:
Could you explain what's going on here? From my knowledge of ffmpeg, this should cause the data stream to be included, right?

Reply:
A `-` before the stream index means deselect.